learning-development/learning-support.qmd

```diff
@@ -65,6 +65,42 @@ Contact [statistics.development@education.gov.uk](mailto:statistics.development@
 
 ---
 
+## Learning and development resources
+
+**Explore education statistics show and tells**
```

Comment:
I'm wondering if we should maybe change the bold titles to level 3 headings so they can be found from the side menu more easily?